packages/api-server/src/watch.ts

```diff
@@ -119,19 +120,19 @@ const rebuildApiServer = () => {
     // Check if experimental server file exists
     const serverFile = resolveFile(`${rwjsPaths.api.dist}/server`)
     if (serverFile) {
-      const separator = chalk.hex('#ff845e')(
-        '------------------------------------------------------------------'
-      )
+      const separator = chalk.hex('#ff845e')('-'.repeat(79))
       console.log(
         [
           separator,
           `🧪 ${chalk.green('Experimental Feature')} 🧪`,
           separator,
-          'Using the experimental API server file at api/dist/server.js',
+          'Using the experimental API server file at api/dist/server.js (in watch mode)',
           separator,
         ].join('\n')
       )
       httpServerProcess = fork(serverFile, [], forkOpts)
+      httpServerProcess.stdout?.pipe?.(process.stdout)
+      httpServerProcess.stderr?.pipe?.(process.stderr)
```

Comment:
i see  logs from the server file coming through without these lines; could you expand a bit more on what wasn't coming through for you?

Reply:
Interesting. I wonder why we get different results.

Here's *with* `.pipe`

![image](https://github.com/redwoodjs/redwood/assets/30793/35b4583e-fd85-44ef-a0f8-2f1c2f4ebfe5)

And here's without

![image](https://github.com/redwoodjs/redwood/assets/30793/b9ec7f28-d4a4-4086-b170-b370fab23a1c)